mewarpx/mewarpx/utils_store/util.py

```diff
@@ -402,3 +402,17 @@ def interpolate_from_grid(coords, grid):
     elif mwxrun.dim == 3:
         raise NotImplementedError("XYZ interpolation not implemented yet.")
     return fpos
+
+
+def mwx_round(x, decimals=0, base=1):
+    """Rounding function that allows rounding around a base value.
+    Arguments:
+        x (float): value to round
+        decimals (int): number of decimal places to round to, defaults to 0.
+            If negative specifies positions left of the decimal point.
+        base (float): base number to round around
+
+    Returns:
+        val (Union[int, float]): rounded value
+    """
+    return np.round(base * np.round(x / base), decimals=decimals)
```

Comment:
If I'm reading this right you're trying to get a multiple of `base`. I don't really understand why `return base * np.round(x / base)` is insufficient in that case. As it is, if `base << 1`, as it is when used in this PR, I find it non-intuitive that this will frequently round to 0 if `decimals` is incorrect.